src/System/executable_query.rs

```diff
@@ -70,6 +75,27 @@ impl ExecutableQuery for RefinementExecutor {
     }
 }
 
+/// Used to store input for the reachability checker
+pub struct ReachabilityExecutor {
+    pub sys: TransitionSystemPtr,
+    pub s_state: State,
+    pub e_state: State,
+}
+
+impl ExecutableQuery for ReachabilityExecutor {
+    fn execute(self: Box<Self>) -> QueryResult {
+        let (sys, s_state, e_state) = (self.sys, self.s_state, self.e_state);
+
+        //match refine::check_refinement(sys1, sys2, sys3) {
+        //Ok(res) => {
+        //    info!("Refinement result: {:?}", res);
+        //    QueryResult::Refinement(res)
+        //}
+        //}
```

Comment:
?

Reply:
It is know removed, since it was not useful. Thanks :)

Reply:
now*

Reply:
@t-lohse friendly reminder =)